pyproject.toml

```diff
@@ -66,6 +68,9 @@ snapshottest = "0.6.0"
 werkzeug = "3.0.1"
 
 
+[tool.poetry.group.dev-dependencies.dependencies]
```

Comment:
Replace the old poetry format on line 42 with this.
Instead of having duplicate groups